Sources/Host/Linux/ARM/PTraceARM.cpp

```diff
@@ -115,26 +115,26 @@ ErrorCode PTrace::readCPUState(ProcessThreadId const &ptid, ProcessInfo const &,
     unsigned long value;
 
     if (wrapPtrace(PTRACE_GETHBPREGS, pid, (n << 1) + 1, &value) < 0) {
-      state.hbp.bcr[n] = 0;
+      state.hbp.bp_addr[n] = 0;
     } else {
-      state.hbp.bcr[n] = value;
+      state.hbp.bp_addr[n] = value;
     }
 
     if (wrapPtrace(PTRACE_GETHBPREGS, pid, (n << 1) + 2, &value) < 0) {
```

Comment:
I wonder why we use a temp variable here (`value`) and we don't for watchpoints?